app/controllers/users_controller.rb

```diff
@@ -55,8 +55,8 @@ def enroll
   # User listing page for Admins                     #
   ####################################################
   def index
-    @users = if params[:email].present?
-               User.search_by_email(params[:email])
+    @users = if params[:email].present? || params[:real_name].present?
+               User.search_by_email_or_real_name(params[:email], params[:real_name])
```

Comment:
This would become an if / elsif / else if we switch to two different query methods on `User`.